packages/block-library/src/cover/index.js

```diff
@@ -81,6 +75,10 @@ export const settings = {
 
 	attributes: blockAttributes,
 
+	supports: {
+		align: [ 'left', 'center', 'right', 'wide', 'full' ],
```

Comment:
I thought making it more explicit would be better in case any more alignments are added in the future, but I'm fine with it being either way.

Reply:
I'd prefer if we were consistent between blocks (I think we're using `true` elsewhere). Also, if we add a new alignment (or remove one), it will update automatically.

Reply:
Alright, I'll change it to `true`, then. :slightly_smiling_face: